imagebuildah/build.go

```diff
@@ -389,12 +389,20 @@ func (b *Executor) Run(run imagebuilder.Run, config docker.Config) error {
 // UnrecognizedInstruction is called when we encounter an instruction that the
 // imagebuilder parser didn't understand.
 func (b *Executor) UnrecognizedInstruction(step *imagebuilder.Step) error {
+	err_str := fmt.Sprintf("Build error: Unknown instruction: %q ", step.Command)
+	err := fmt.Sprintf(err_str+"%#v", step)
```

Comment:
These lines should be combined.

err := fmt.Sprintf("Build error: Unknown instruction: %q  %#v", ste.Command, step)

Reply:
They are different. One is giving the full step info. The other is trying to simplify for less verbose output.

Reply:
Ok you are using them in different output commands. Never mind.